build_tools/packaging/wheel/osx_wheel.sh

```diff
@@ -1,7 +1,7 @@
-if [[ ":$PATH:" == *"conda"* ]]; then
```

Comment:
without this, you are prone to accidents. but up to you.

Reply:
I am not sure how the accident occurs; the script initializes a fresh conda environment and then sources it?

Reply:
if you already have a conda install in path, and you install + source a new conda install, the PATH will have both conda installs at the same time. Then your cmake will find stuff in the first OR the second conda install

Reply:
I see. OK I'll bring it back